tests/server/test_loggers.py

```diff
@@ -54,6 +54,44 @@ def test_default_logger():
     assert isinstance(server_logger.logger.loggers[0].logger.loggers[0], PythonLogger)
 
 
+def test_data_logging_from_predefined():
+    server_config = ServerConfig(
+        endpoints=[
+            EndpointConfig(
+                task=task,
+                name="text_classification",
+                model=stub,
+                add_predefined=[MetricFunctionConfig(func="text_classification")],
+            )
+        ],
+        loggers={"logger_1": {"path": logger_identifier}},
+    )
+    server_logger = server_logger_from_config(server_config)
+    with mock.patch(
+        "deepsparse.server.server.server_logger_from_config", return_value=server_logger
+    ), mock_engine(rng_seed=0):
+        app = _build_app(server_config)
+    client = TestClient(app)
+    client.post(
+        "/predict",
+        json={
+            "sequences": [["Fun for adults and children.", "Fun for only children."]]
+        },
+    )
+    data_logging_logs = [
+        call
+        for call in server_logger.logger.loggers[0].logger.loggers[0].calls
```

Comment:
this is gonna need at least a comment, or to be moved to a variable with a readable name :)